tidb-cloud/built-in-monitoring.md

```diff
@@ -32,6 +28,10 @@ All monitoring data is kept for 2 days.
 
 The following sections illustrate the metrics on the Monitoring page.
 
+> **Note:**
+>
+> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides a subset of monitoring metrics, which are marked with `*` in the **Metric name** column of the following sections.
```

Comment:
How about this? 
```suggestion
> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides a subset of monitoring metrics, which are marked with `*` in the **Metric name**.
```

Reply:
As the actual metric name does not include `*` in it, can we keep "in the **Metric name** column" and remove "of the following sections"?

Reply:
Sure. Updated.